src/app/shared/constants.ts

```diff
@@ -0,0 +1,6 @@
+export class Constants {
+  /**
+   * Max length 255 for some entities fields
+   */
+  public static readonly maxLength255 = 255;
```

Comment:
I guess It's a bad constant naming example. Imagine that you will need to change `255` to `256` it will lead to changes in constant name, which leads to changes in other files. 

The key idea of such constants - is to reduce the number of places which requires changes in the case of constant value changes.